### PR TITLE
test: add email submission e2e tests

### PR DESCRIPTION
**Pull request by karrui**
More tests, trying to achieve parity (and then some) to the current e2e tests

### CODE REVIEW

**Comment by justynoh** [comment]
can't request for kr to review cuz he's the PR owner but @karrui 🙏

File: __tests__/e2e/utils/database.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    }
+  }
+
+  // Need this try catch block as some schemas may have been converted to
+  // TypeScript and use default exports instead.
+  try {
+    return spec(`dist/backend/src/app/models/${modelFilename}`)(db)
+  } catch (e) {
+    return spec(`dist/backend/src/app/models/${modelFilename}`).default(db)
+  }
```
Comment (karrui) [inline __tests__/e2e/utils/database.ts]: will not need try catch block, everything is now typescript

File: __tests__/e2e/utils/database.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import mongoose, { Model } from 'mongoose'
```
Comment (karrui) [inline __tests__/e2e/utils/database.ts:1]: do we need this file at all? maybe just need the mongoose connection? I think the rest should work on its own without these functions...

**Review by karrui — COMMENTED** [review]
lgtm but i can't approve my own PR lol

File: __tests__/e2e/utils/database.ts (hunk 1 above)
Comment (justynoh) [reply __tests__/e2e/utils/database.ts]: [54ac427](https://github.com/opengovsg/FormSG/pull/5162/commits/54ac427af191a751807ee590f6a857d81e8f1a7d)

File: __tests__/e2e/utils/database.ts (hunk 2 above)
Comment (justynoh) [reply __tests__/e2e/utils/database.ts:1]: as discussed, can keep form collection

**Comment by justynoh** [comment]
passing, but some tests still flaky. will work on fixing this before merging

**Review by justynoh — APPROVED** [review]
approving as reviewed by kr!

**Pull request merged**